frontend/apps/pwa/src/components/formFields/MultilineTextFieldEditor.tsx

```diff
@@ -1,24 +1,30 @@
 import React, { FC } from 'react';
 
 import { FieldEditorProps } from './types';
-import { FieldDescription } from './FieldDescription';
-import { FieldLabel } from './FieldLabel';
 
 export const MultilineTextFieldEditor: FC<FieldEditorProps> = ({
   field,
   value,
   onChange,
+  register,
+  errors,
 }) => {
+  const registerObject = register(`values.${field.id}`, {
+    required: { value: field.required, message: 'This field is required' },
+  });
   return (
-    <div className="form-group mb-2">
-      <FieldLabel fieldDefinition={field} />
-      <textarea
-        className="form-control bg-dark text-light border-secondary"
-        id={field.id}
-        value={value || ''}
-        onChange={(event) => onChange(event.target.value)}
-      />
-      <FieldDescription text={field.description} />
-    </div>
+    <textarea
+      className={`form-control bg-dark text-light border-secondary ${
+        errors?.values && errors?.values[field.id] ? 'is-invalid' : ''
+      }`}
+      id={field.id}
+      value={value || ''}
+      {...registerObject}
+      onChange={(event) => {
+        onChange(event.target.value);
+        return registerObject.onChange(event);
+      }}
+      aria-describedby="errorMessages"
```

Comment:
Should this not be the id of the description field? I think it can have multiple entries so we could include both.